dist/runner/variant-reassignment.js

```diff
@@ -0,0 +1,922 @@
+'use strict';
```

Comment:
Do we need generated dist directory here?

Reply:
It's for testing only, because right now I don't have npm repository and I get this lib into sphere-product-import using github link. I hope when I have the npm repo, then I don't have to commit dist folder anymore, but the dist folder will be created when building the project.